modules/local/hicpro/bowtie2_merge.nf

```diff
@@ -1,8 +1,8 @@
 process MERGE_BOWTIE2{
```

Comment:
This could be named `SAMTOOLS_MERGE_SORT`

Reply:
But I can see all the extra stuff you're doing for BOWTIE2.
Just confused about the `hicpro` folder

Reply:
I'll keep it like that for this release

Reply:
No problem at all, it really was the less important of my comments